planetary_computer/sas.py

```diff
@@ -84,6 +91,13 @@ def sign_url(url: str) -> str:
     Returns:
         str: The signed HREF
     """
+    if is_vrt_string(url):
+        return sign_vrt_string(url)
+    else:
+        return sign_url(url)
+
+
+def sign_url(url: str) -> str:
```

Comment:
Needs a docstring.

Reply:
And I wonder if the implementation can / should use the regex-based implementation required by VRTs. I'll need to time it.